brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Masonary - Fixed the JS UAGBPostMasonry is not defined error.

### Description

Trello - https://trello.com/c/RrMa2Lit/278-fix-post-masonry-fixed-the-js-uagbpostmasonry-is-not-defined-error

### Screenshots

Step to reproduce -
- Activate only Masonry block.
- Drag and drop post masonry update the page and check the console on the front-end.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate only Masonry block.
- Drag and drop post masonry update the page and check the console on the front-end.
- Activate all blocks and check the console again.

### Checklist:
